recipes-bsp/u-boot/u-boot-imx_2017.03.bb

```diff
@@ -20,5 +20,29 @@ inherit dtc-145 fsl-u-boot-localversion
 
 LOCALVERSION ?= "-${SRCBRANCH}"
 
+BOOT_TOOLS = "imx-boot-tools"
+
+do_deploy_append_mx8mq () {
+    # Deploy the mkimage, u-boot-nodtb.bin and fsl-imx8mq-XX.dtb for mkimage to generate boot binary
```

Comment:
Why would someone need to recreate the U-Boot blob from the deploy files? I fail to see the use-case.

This is an important point to be discussed as multiple SoCs already make use of DeviceTree in U-Boot but none has had the need to put them inside the deploy directory. 

Is it possible for you to explain the use-case of it?

Reply:
I was not a part of this design. Can you ask Jun? I assume you see that these artifacts are gathered in the imx-boot compile stage. I had the impression it was something to do about gathering files for the boot partition. I also had the impression that there might be existing mechanisms for accomplishing this instead of this design we have.